ui/app/components/clients/page/counts.hbs

```diff
@@ -58,49 +57,51 @@
-            <SearchSelect
-              @id="namespace-search-select"
-              @options={{this.namespaces}}
-              @inputValue={{if @namespace (array @namespace)}}
-              @selectLimit="1"
-              @disallowNewItems={{true}}
-              @fallbackComponent="input-search"
-              @onChange={{fn this.setFilterValue "ns"}}
-              @placeholder="Filter by namespace"
-              @displayInherit={{true}}
-              class="is-marginless"
-              data-test-counts-namespaces
+    {{#if @startTimestamp}}
+      <div class="is-subtitle-gray has-bottom-margin-m">
+        FILTERS
+        <Toolbar data-test-clients-filter-bar>
+          <ToolbarFilters>
+            <CalendarWidget
+              @startTimestamp={{this.startTimeISO}}
+              @endTimestamp={{this.endTimeISO}}
+              @selectMonth={{this.onDateChange}}
             />
-          {{/if}}
-          {{#if (or @authMount this.authMounts)}}
-            <SearchSelect
-              @id="auth-method-search-select"
-              @options={{this.authMounts}}
-              @inputValue={{if @authMount (array @authMount)}}
-              @selectLimit="1"
-              @disallowNewItems={{true}}
-              @fallbackComponent="input-search"
-              @onChange={{fn this.setFilterValue "authMount"}}
-              @placeholder="Filter by auth method"
-              @displayInherit={{true}}
-              data-test-counts-auth-mounts
-            />
-          {{/if}}
-        </ToolbarFilters>
-      </Toolbar>
-    </div>
+            {{#if (or @namespace this.namespaces)}}
+              <SearchSelect
+                @id="namespace-search-select"
+                @options={{this.namespaces}}
+                @inputValue={{if @namespace (array @namespace)}}
+                @selectLimit="1"
+                @disallowNewItems={{true}}
+                @fallbackComponent="input-search"
+                @onChange={{fn this.setFilterValue "ns"}}
+                @placeholder="Filter by namespace"
+                @displayInherit={{true}}
+                class="is-marginless"
+                data-test-counts-namespaces
+              />
+            {{/if}}
+            {{#if (or @mountPath this.mountPaths)}}
+              <SearchSelect
+                @id="auth-method-search-select"
+                @options={{this.mountPaths}}
+                @inputValue={{if @mountPath (array @mountPath)}}
+                @selectLimit="1"
+                @disallowNewItems={{true}}
+                @fallbackComponent="input-search"
+                @onChange={{fn this.setFilterValue "mountPath"}}
+                @placeholder="Filter by mount path"
+                @displayInherit={{true}}
+                data-test-counts-auth-mounts
+              />
+            {{/if}}
+          </ToolbarFilters>
+        </Toolbar>
+      </div>
+    {{/if}}
 
-    {{#if this.filteredActivity}}
+    {{#if this.totalUsageCounts}}
```

Comment:
I originally added `filteredActivity` in an attempt to only render the child routes if there was activity. Will `totalUsageCounts` provide the same thing?